ghcide/src/Development/IDE/GHC/Error.hs

```diff
@@ -79,7 +79,7 @@ realSrcSpanToRange real =
 
 realSrcLocToPosition :: RealSrcLoc -> Position
 realSrcLocToPosition real =
-  Position (srcLocLine real - 1) (srcLocCol real - 1)
+  Position (fromIntegral $ srcLocLine real - 1) (fromIntegral $ srcLocCol real - 1)
```

Comment:
This will currently wrap. Should we do something else here? Like take `max 0` first?

Reply:
yeah not sure if we have another sensible alternative?

Reply:
Not really. I think either let it wrap or max with 0.